tests/compile-fail/function_calls/exported_symbol_bad_unwind2.rs

```diff
@@ -14,6 +16,4 @@ fn main() {
     }
     unsafe { nounwind() }
     //[extern_block]~^ ERROR unwinding past a stack frame that does not allow unwinding
```

Comment:
With `extern_block`, there is no definition, right? Why does this not show an error about an unknown extern function being called...?

Reply:
I think the definition is present even with `extern_block`, it just does not have `#[rustc_allocator_nounwind]`.

Reply:
Oh, right, it's `cfg_attr` not `cfg`... :)